portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,70 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.time.format.DateTimeFormatter;  
+import java.time.ZonedDateTime;   
+import java.time.ZoneId;   
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    final private ArrayList<Comment> allComments = new ArrayList<Comment>();
+
+    /**
+    * Class to organize comments for more detailed displays. By creating 
+    * another class, adding new parameters and information to keep track 
+    * of may be easier.
+    */
+    private static class Comment {
+        private String name;
+        private String comment;
+        private ZonedDateTime date;
+
+        Comment(String newName, String newComment) {
+            name = newName;
+            comment = newComment;
+            date = getDateTime(); 
+        }
+
+        /** @return a formatted string with all of the comment information. */
```

Comment:
is this used anywhere? it could be nice to have for debugging, so i'm not against it - but generally we want to avoid commiting dead code.

Reply:
Removed.